nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/TeamAutomationHandler.java

```diff
@@ -64,21 +64,40 @@ public void createNewTeam(TeamEntity newTeamEntity) {
                 new TeamMemberId(newTeamEntity.getCreatedBy().getMemberId()),
                 teamSlackIdentity);
 
+        AtomistMemberBase atomistMemberBase = new AtomistMemberBase(
+                teamMemberEntity.getFirstName(),
+                teamMemberEntity.getDomainUsername(),
+                teamMemberSlackIdentity);
+
         TeamCreatedWithDetails newTeam = new TeamCreatedWithDetails(teamCreated,
-                new AtomistMemberBase(
-                        teamMemberEntity.getFirstName(),
-                        teamMemberEntity.getDomainUsername(),
-                        teamMemberSlackIdentity));
+                atomistMemberBase);
 
         ResponseEntity<String> response = restTemplate.postForEntity(
                 atomistConfigurationProperties.getTeamCreatedEventUrl(),
                 newTeam,
                 String.class);
 
         if (response.getStatusCode().is2xxSuccessful()) {
-            log.info("Atomist has ingested event successfully: {} -> {}",
+            log.info("Atomist has ingested teamCreatedEvent successfully: {} -> {}",
```

Comment:
I would name this using same casing as in Atomist ingester.

```suggestion
            log.info("Atomist has ingested TeamCreatedEvent successfully: {} -> {}",
```